src/pages/workspace/accounting/xero/export/XeroExportConfigurationPage.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import ConnectionLayout from '@components/ConnectionLayout';
+import type {MenuItemProps} from '@components/MenuItem';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import OfflineWithFeedback from '@components/OfflineWithFeedback';
+import type {OfflineWithFeedbackProps} from '@components/OfflineWithFeedback';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import type {WithPolicyConnectionsProps} from '@pages/workspace/withPolicyConnections';
+import withPolicyConnections from '@pages/workspace/withPolicyConnections';
+import CONST from '@src/CONST';
+
+type MenuItem = MenuItemProps & {pendingAction?: OfflineWithFeedbackProps['pendingAction']};
+
+function XeroExportConfigurationPage({policy}: WithPolicyConnectionsProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const policyID = policy?.id ?? '';
+    const policyOwner = policy?.owner ?? '';
+
+    const {export: exportConfiguration, errorFields, pendingFields} = policy?.connections?.xero?.config ?? {};
+    const menuItems: MenuItem[] = [
+        {
+            description: translate('workspace.xero.preferredExporter'),
+            onPress: () => {},
+            brickRoadIndicator: errorFields?.exporter ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
+            title: exportConfiguration?.exporter ?? policyOwner,
+            pendingAction: pendingFields?.export,
+            error: errorFields?.exporter ? translate('common.genericErrorMessage') : undefined,
+        },
+        {
+            description: translate('workspace.xero.exportExpenses'),
+            title: translate('workspace.xero.purchaseBill'),
+            interactive: false,
+            shouldShowRightIcon: false,
+            helperText: translate('workspace.xero.exportExpensesDescription'),
+        },
+        {
+            description: translate('workspace.xero.purchaseBillDate'),
+            brickRoadIndicator: errorFields?.billDate ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
+            title: exportConfiguration?.billDate,
+            pendingAction: pendingFields?.export,
+            error: errorFields?.billDate ? translate('common.genericErrorMessage') : undefined,
+        },
+        {
+            description: translate('workspace.xero.advancedConfig.purchaseBillStatusTitle'),
+            onPress: () => {},
+            title: exportConfiguration?.billStatus.purchase,
```

Comment:
I got crashed on IOS, Android on this line, so I think it'd better like this:
```suggestion
            title: exportConfiguration?.billStatus?.purchase,
```